src/main/resources/sql/dummy-data/README.md

```
@@ -0,0 +1,186 @@
+
+-- 3. AI 모델 데이터 생성  
+SOURCE 03_ai_models.sql;
+
+-- 4. 모델 통계 데이터 생성
+SOURCE 04_model_statistics.sql;
+
+-- 5. 구독 상품 데이터 생성
+SOURCE 05_subscriptions.sql;
+```
+
+### 터미널에서 실행
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)